docs/content/manual/release-specific/v1.1.0/rwx_feature.md

```diff
@@ -0,0 +1,320 @@
+---
+title: Test Read Write Many Feature
+---
+
+# Prerequisite:
+1. Set up a Cluster of 4 nodes (1 etc/control plane and 3 workers)
+2. Deploy Latest Longhorn-master
+
+# Create StatefulSet/Deployment with single pod with volume attached in RWX mode.
+1. Create a StatefulSet/Deployment with 1 pod.
+2. Attach a volume with RWX mode using longhorn class and selecting the option `read write many`.
+3. Verify that a PVC, ShareManger pod, CRD and volume in Longhorn get created.
+4. Verify there is directory with the name of PVC exists in the ShareManager mount point.
```

Comment:
need to check for share-manager pod ready, which is the signal once the volume is attached, mounted and the nfs-server is up.

Reply:
Done.